libcontainer/integration/template_test.go

```diff
@@ -213,9 +214,9 @@ func newTemplateConfig(p *tParam) *configs.Config {
 	}
 
 	if p.systemd {
-		id := strconv.FormatUint(rand.Uint64(), 36)
-		config.Cgroups.Name = "test" + id
-		// do not change Parent (see newContainerWithName)
+		id := strconv.FormatInt(-int64(time.Now().Nanosecond()), 36)
```

Comment:
minus~ good one

Reply:
Why not just use `rand.Uint64` to prevent from using the same id

Reply:
Because it needs to be seeded somewhere